### DISCUSSION TITLE
知道共产党最无耻的地方在哪里吗？

### DISCUSSION THREAD

**Discussion by freelandcn** [WeNeedHome/SummaryOfLoanSuspension#963] [General]
就是把人民当肉盾！

2019年的时候，一个认识的90后同事娶老婆，要买房，平时穷得到处网贷，被女人逼得我看着他都焦虑，后来不知道刷了多少信用卡来付了首付，每个月连加油的钱都没有。

我当时就感觉，最后一棒了……但是我又能跟他说什么吗？说共产党这几年知道接下来经济要完蛋，这三年拼命拉高房价，就是要让你们接盘？我这么说，他也不会听，因为他不得不买房，为了能讨个老婆，他就算赌命也会买的。

我不知道后来他的房子烂尾了没，很巧，他也是河南人。

你们小时候看着那些抗日剧，共产党多么英勇，打日本人，都是“兄弟们，跟我来”，国民党的领导们则是“兄弟们，给我上”。现在这把年纪了，终于明白了吧，那都是拍戏。2016年可是财政部亲儿子信达地产身先士卒到处拍地王哦，所以这次的暴雷，根本就不是什么意外事件，什么恒大之流，早就知道自己拼命加杠杆只有一个后果，那就是烂尾——只不过烂在谁手上区别大了。

所以，如果你们明白背后的逻辑，就会知道，最终的解决方案必然是一个字”拖“。能拖多久拖多久，软硬兼施、分化打击、栽赃诬陷，总之没有底线的玩法要多少有多少，就是一个目标，让你们成为——成就光荣的汉武大帝的”代价“。

对的，这个”代价“，直接来看，就是你们，间接来看，是中国权贵阶层以外的所有人。

最后，这个项目页上写的那些”相信XX“，真的不是反讽吗？程序员们，你们不是最讲逻辑的群体吗？这叫有逻辑？我看这就是轻视文科，不了解真实历史的代价吧。

**Comment by freelandcn** [comment]
修正一下最后一句，我想在被谎言洗脑了无数遍之后，各位都是受害者，和文理无关。

不管是香港、台湾、中日、中美、8964、文革，所有这些历史事件，你如果自认为和那些U形锁的草履虫大脑有所不同的话，先问问你自己，当新闻里提到那些名词的时候，你那满满的厌恶，到底是自从3岁起外界灌输给你的呢？还是你自己像个历史学家去研究史料、多方印证、不偏听任何一方说辞，通过合理推论而得到的历史真相继而产生的情绪呢？

他们跟你说他们是民主的，然后定义了全过程民主。
他们跟你说他们是科学的，然后污名化专家，封了一些专家的号，放出另一些愿意按他们意志来辩护的专家。
他们跟你说他们是依法治国，然后你不服的时候，他们随时修改法律告诉你，领导的看法已经写进法律，你服不服。

你要不是傻子的话，你还信他们？

**Comment by NiceAir** [comment]
想发泄情绪没问题，可这个项目不是让你来发泄情绪的，找错地方了

**Reply by hrsgn** [reply]
哪里说得不对，请反驳，别上来就扣发泄情绪的帽子

**Reply by clleo888** [reply]
人家在讲事实, 你说人家发泄情绪, 我看你才是在发泄情绪。
伤害了你那个骗了你无数次, 还一直迫你爱它的党了吗?

**Reply by ghost** [reply]
真是你党孝子

**Comment by zongjiangU** [comment]
> 项目页上写的相信党、相信政府，可以肯定的是这不是一句真心话。国内高压的舆论管制已经将恐惧刻入了绝大数人的内心深处，这么写会有心理安慰与心理暗示，不会被查水表。个人觉得这么写无可厚非，能理解，但也不赞同。
 
>  不过说党会倒的，我认为几乎不可能，一亿的党员，党政不分离，真正的树大不倒。最有可能的也就是政改了，但是谁又真的能在权贵手里夺权呢，美国不能，北约不能，整个世界也不能。外部力量会让权贵内部意见一致，具有核打击的军事能力，外部是无法撼动的。

> 这件事情，只能从内部发起，形成瓦解。破局者只能是万千百姓。维权的群众多了，迫于政治压力，定会做出拆东墙补西墙的决定。而这个拆的动作，会让盘子里的蛋糕分配不均，进而形成更多小利益集团之间对抗。

> 最重要的一点是，这依赖一位技术官僚出身的人执掌江山来推动改革，绝不能是二代三代，深陷利益之争的人。但此人又必须有足够的能量推行政改。似乎与利益形成悖论，但坚信会有这么一天，会有这么一个人出现的。

> 最后对那些`幻想欧美能破防共产党的人，幻想所谓加速能到瓦解的人` 说，别做白日梦了，手握核打击能力，就是在边防栓条狗，欧美也不敢越它分毫。那些在各个论坛打着汉字评论称呼中国人为`支`的人,你们真的连畜生都不如，那是侵华日军对中国人的蔑称，战后日本都禁止使用这个词了，你还拿出来用，不是傻就是真的没了良心。

希望头顶的天真的会晴朗，希望中国真正的越来越好

**Reply by Elijah-Wan** [reply]
1981年的苏共也是这么想的

**Reply by CKwuliuqi** [reply]
每一个王朝的末年，都是春江水暖鸭先知。

**Comment by jqknono** [comment]


**Reply by mand-byte** [reply]
资本世界可以骂政府，你能吗，你先搞懂资本世界再喷

**Reply by ZerodLee** [reply]
> 资本世界可以骂政府，你能吗，你先搞懂资本世界再喷

咋滴，更愿意相信收割一切的大资本，被他们管着？恐怕只会让你在大资本财阀手里永无出头之日

**Reply by 378804441** [reply]
> 
😂 就这点格局么？为了骂一句政府我把我家点着了？
我们指向吃顿红烧肉，但是某些人啊，想借着给我做顿红烧肉的名义把我家给点着了。

**Reply by itsoursduty** [reply]
@378804441 你没意识到的是 不把家点了重新建，当家作主的人这辈子都不会给你红烧肉吃，还指望那些人把嘴里的红烧肉分给屁民吃？就这点格局么？

**Reply by mand-byte** [reply]
> > 资本世界可以骂政府，你能吗，你先搞懂资本世界再喷
> 
> 咋滴，更愿意相信收割一切的大资本，被他们管着？恐怕只会让你在大资本财阀手里永无出头之日

手机就是大资本创新发明的，别用啊，吃完了骂娘？用共产党发明的东西去

**Reply by ZerodLee** [reply]
> > > 资本世界可以骂政府，你能吗，你先搞懂资本世界再喷
> > 
> > 
> > 咋滴，更愿意相信收割一切的大资本，被他们管着？恐怕只会让你在大资本财阀手里永无出头之日
> 
> 手机就是大资本创新发明的，别用啊，吃完了骂娘？用共产党发明的东西去

你看看你说的内容有内在逻辑吗，被某种意识形态冲昏了就是你这样的，不想骂的

**Comment by redapple2021** [comment]
> 就是把人民当肉盾！
> 
> 2019年的时候，一个认识的90后同事娶老婆，要买房，平时穷得到处网贷，被女人逼得我看着他都焦虑，后来不知道刷了多少信用卡来付了首付，每个月连加油的钱都没有。
> 
> 我当时就感觉，最后一棒了……但是我又能跟他说什么吗？说共产党这几年知道接下来经济要完蛋，这三年拼命拉高房价，就是要让你们接盘？我这么说，他也不会听，因为他不得不买房，为了能讨个老婆，他就算赌命也会买的。
> 
> 我不知道后来他的房子烂尾了没，很巧，他也是河南人。
> 
> 你们小时候看着那些抗日剧，共产党多么英勇，打日本人，都是“兄弟们，跟我来”，国民党的领导们则是“兄弟们，给我上”。现在这把年纪了，终于明白了吧，那都是拍戏。2016年可是财政部亲儿子信达地产身先士卒到处拍地王哦，所以这次的暴雷，根本就不是什么意外事件，什么恒大之流，早就知道自己拼命加杠杆只有一个后果，那就是烂尾——只不过烂在谁手上区别大了。
> 
> 所以，如果你们明白背后的逻辑，就会知道，最终的解决方案必然是一个字”拖“。能拖多久拖多久，软硬兼施、分化打击、栽赃诬陷，总之没有底线的玩法要多少有多少，就是一个目标，让你们成为——成就光荣的汉武大帝的”代价“。
> 
> 对的，这个”代价“，直接来看，就是你们，间接来看，是中国权贵阶层以外的所有人。
> 
> 最后，这个项目页上写的那些”相信XX“，真的不是反讽吗？程序员们，你们不是最讲逻辑的群体吗？这叫有逻辑？我看这就是轻视文科，不了解真实历史的代价吧。

我感觉现在那些95往后和50岁往上的中党的毒更深，没有逻辑性，很悲哀！

**Reply by JerryPlayer123** [reply]
60岁往上吧，五十岁还是有不少理智的人的

**Reply by maxwillma** [reply]
现在中二遍地走，教育这个公器被私用了

**Comment by bfsh201905** [comment]
以前我也相信党、相信政府。我现在不相信了，买了房之后才发现开发商和政府才是一个世界的，买完房之后我就被政府定义在对立面了。地产商照成的问题，结果都是我在承担，我合法维权都进局子几次了，我们的维权行为最终都成为人家的饭后谈资。

**Reply by ytx0574** [reply]
有老哥负重前行, 咱也得努力行动

**Reply by CKwuliuqi** [reply]
对啊，没经历过铁拳的人，永远不明白铁拳砸在身上有多痛，痛醒了还无能为力，是更深沉的悲伤

**Reply by maxwillma** [reply]
政府监管部门没责任，银行没责任，开发商没责任，只有买了烂尾楼的屁民有责任。

**Comment by JusticeEER** [comment]
共产党的话不要信就行了，说不听的人不用理，对于那些不敢讨伐共产党而又喜欢欺负普通人的人直接干他就行，我不信那些共产党或者支持共产党的人没有父母没有子女的，他让你征信影响后代，你也可以对付他们的后代，现在开盒不难
对于有些之前超级粉红的现在被铁拳打的，可以帮他们开盒然后可以叫他们献忠

**Comment by GitHeP** [comment]
除非他们觉醒，否则永远不会反抗，但除非他们反抗，否则不会觉醒。《1984》

**Comment by aiheihe** [comment]
又当又立  婊子

**Reply by TomMao-CN** [reply]
从大家的点赞数量可以看出来，谁是谁非

**Comment by jbl19860422** [comment]
习近平还想终身制，这个傻逼包子，连字都不会读，以后农民和底层有苦日子过了

**Reply by xijinping0** [reply]
> 习近平在入学前是初中尚未毕业，一下子要进入大学深造，的确是有点勉为其难。在刚入学的数学摸底考试中，习近平就闹了一个大笑话。测试的题目是：三分之一加五分之一等于多少？习近平的答案是八分之二。当老师用习的回答作为案例来讲解时，顿时引来教室内的同学的哄堂大笑。可习近平还是展示了他日后的沉稳心理，他微微一笑对老师说：“老师，八分之二的答案是错误的，因为我的确没有学过，那是昨天，但是今天我知道正确答案是十五分之八，我也了解了最小公约数和通分的规则。”老师很惊奇问她，你是如何学习的？习近平笑呵呵向他的同桌的陈希点点头。陈希向老师介绍了习近平昨天考试后如何向他请教如何演算分数。习只花了一晚上就把分数的四则运算全部掌握，所以今天他就很坦然了。若干年后，陈希在回忆这段往事时十分感慨，他对习近平的聪明和谦虚态度感到十分佩服。

--[《习近平与他的情人们》](https://xijinping0.github.io/lovers/docs/toc)

**Comment by naifenmizuha** [comment]
怪不得GitHub一直在**墙边上**

**Comment by Dreaming-Pepper** [comment]
这是国策，以性驭男，毕竟郭楠是世界上少有不反抗，不反思，勤劳、PUA自己的人类，国家当然不会放过这么好的牲口

**Reply by Dreaming-Pepper** [reply]
但凡思想解放，Free mind 后，都不至于活得那么悲催

**Comment by ZerodLee** [comment]
你首先要知道，任何国家管理者都想维护自己的统治地位，其手段各有不同，包装程度也不尽相同，所以就有不同的社会制度。你喷中共，因为你只知道中共，而且还是表面的，当权者愿意给你看的，凭这些信息去反这反那，只是一个拙劣喷子的水平。

**Reply by itsoursduty** [reply]
所以你是不是该指条明路

**Comment by Cegazhos** [comment]
说烂尾就说烂尾，人家停贷项目是希望能收房的 不是给你舞台输出的。
我也不喜欢一些东西，但是为什么要在别人的项目上发泄呢？

**Reply by mand-byte** [reply]
找bug不是找源头，你要么智商有问题，要么还是改行别做程序员，别在github上拉屎了

**Reply by Cegazhos** [reply]
有能力改这个源头BUG的程序员是你么？既没有修改权限，又没有修改能力，为什么非要借势输出？
你家没有厕所么 为什么到别人的项目上拉屎？

**Reply by mand-byte** [reply]
既然你认为没能力你就闭嘴，让有能力的上，让更多的人知道，请闭上你的嘴

**Reply by Cegazhos** [reply]
> 既然你认为没能力你就闭嘴，让有能力的上，让更多的人知道，请闭上你的嘴

你着急的样子真可爱🥰

**Reply by ghost** [reply]
> 既然你认为没能力你就闭嘴，让有能力的上，让更多的人知道，请闭上你的嘴

你他吗的笑死我了，你看看你那飞舞样，nm比都没你能叫🤣，赶紧去吃你妈的屎吧,cnm的。

**Reply by tanuk1997** [reply]
> > 既然你认为没能力你就闭嘴，让有能力的上，让更多的人知道，请闭上你的嘴
> 
> 你他吗的笑死我了，你看看你那飞舞样，nm比都没你能叫🤣，赶紧去吃你妈的屎吧,cnm的。

赶紧和你妈做核酸去吧，一脸奴才样🤣

**Comment by JayChou55** [comment]
我是李文亮时期清醒的。。。已经死了，然后拉出来各种抢救 各种操作，人挂了都不放过

**Reply by Cegazhos** [reply]
> 我是李文亮时期清醒的。。。已经死了，然后拉出来各种抢救 各种操作，人挂了都不放过

台湾佬能不能别几把反串了啊？演技这么差 不带一点脑 你怎么好意思装清醒的？怎么不多吃点莱猪脑治一治啊？

**Comment by a6749920** [comment]
唐山鲜明的例子，发生在你我身上是什么样的感觉

**Reply by Cegazhos** [reply]
烧烤店打人不够等级，怎么扯也是底层人的治安事件。710这类才是严重的官压民

**Reply by summerBreeze03** [reply]
唐山就是拿来混淆视听的，本来一件芝麻小事儿，被拿去当挡箭牌了

**Reply by maxwillma** [reply]
> 唐山就是拿来混淆视听的，本来一件芝麻小事儿，被拿去当挡箭牌了

明星绯闻也同样的作用

**Comment by a6749920** [comment]
在我们这里建垃圾站，不同意，派了武警镇压，给了一点拆迁费

**Comment by wangsir003** [comment]
理一下你的逻辑就发现没有逻辑，为了扣屎盆而已，多读书吧

**Comment by lance111l** [comment]
小编智商不高  被小公司骗了  自己不反省  在这骂政府  
到哪个国家都要有头脑  不然还会上当受骗

**Comment by summerBreeze03** [comment]
放心吧，这是这片土地的问题，不是党的问题，从古至今都这样，已经两千年了

**Reply by maxwillma** [reply]
都怪资本家 ：）

**Comment by hooyantsing** [comment]
七分发展、两分应付、一分抗日

**Comment by b4o** [comment]
随你怎么叫嚣抹黑，对中国的再次崛起没有丝毫影响，你还是担心下自己吧，将来某一天美国资本家为了自己利益，和中国政府合作，到时候全球通缉你们，我是说真的，美国政治以钱为中心，中国政府用钱就可以解决，你们还是好好担心自己的未来吧

**Reply by maxwillma** [reply]
浓浓的暴发户味道

**Comment by summerBreeze03** [comment]
这么说吧，我妈很爱我，她绝不会包庇我的过错，当我表现好，考试考了第一的时候她绝不吝啬夸赞；当我淘气做了坏事的时候，她也不吝责罚。现在我当然很爱国，你也是，那么请问当国家犯错的时候，你应该怎么做？获取Outlook for Android发件人: ***@***.*** ***@***.***> 代表 kylin ***@***.***>发送时间: 星期五, 六月 9, 2023 5:21:48 下午收件人: SummaryOfLoanSuspension ***@***.***>抄送: 2538999495 ***@***.***>; comment ***@***.***>主题: Re: [WeNeedHome/SummaryOfLoanSuspension] 知道共产党最无耻的地方在哪里吗？ (Discussion #963)
随你怎么叫嚣抹黑，对中国的再次崛起没有丝毫影响，你还是担心下自己吧，将来某一天美国资本家为了自己利益，和中国政府合作，到时候全球通缉你们，我是说真的，美国政治以钱为中心，中国政府用钱就可以解决，你们还是好好担心自己的未来吧

**Comment by ghost** [comment]
> 就是把人民当肉盾！
> 
> 2019年的时候，一个认识的90后同事娶老婆，要买房，平时穷得到处网贷，被女人逼得我看着他都焦虑，后来不知道刷了多少信用卡来付了首付，每个月连加油的钱都没有。
> 
> 我当时就感觉，最后一棒了……但是我又能跟他说什么吗？说共产党这几年知道接下来经济要完蛋，这三年拼命拉高房价，就是要让你们接盘？我这么说，他也不会听，因为他不得不买房，为了能讨个老婆，他就算赌命也会买的。
> 
> 我不知道后来他的房子烂尾了没，很巧，他也是河南人。
> 
> 你们小时候看着那些抗日剧，共产党多么英勇，打日本人，都是“兄弟们，跟我来”，国民党的领导们则是“兄弟们，给我上”。现在这把年纪了，终于明白了吧，那都是拍戏。2016年可是财政部亲儿子信达地产身先士卒到处拍地王哦，所以这次的暴雷，根本就不是什么意外事件，什么恒大之流，早就知道自己拼命加杠杆只有一个后果，那就是烂尾——只不过烂在谁手上区别大了。
> 
> 所以，如果你们明白背后的逻辑，就会知道，最终的解决方案必然是一个字”拖“。能拖多久拖多久，软硬兼施、分化打击、栽赃诬陷，总之没有底线的玩法要多少有多少，就是一个目标，让你们成为——成就光荣的汉武大帝的”代价“。
> 
> 对的，这个”代价“，直接来看，就是你们，间接来看，是中国权贵阶层以外的所有人。
> 
> 最后，这个项目页上写的那些”相信XX“，真的不是反讽吗？程序员们，你们不是最讲逻辑的群体吗？这叫有逻辑？我看这就是轻视文科，不了解真实历史的代价吧。

我给你一个建议，你没逻辑的地方在于。为什么一个党说自己叫共产党，你就觉得他是共产党。
那我们还有（德国）社会主义工人党，这个是共产党吗？
